hash/risc0/bench/host/benches/bench_main.rs

```diff
@@ -0,0 +1,44 @@
+use criterion::Criterion;
+// use clap::Parser;
+// use std::env;
+mod benchmarks;
+
+fn main() {
+    let mut criterion: criterion::Criterion<_>  = (Criterion::default().sample_size(10)).configure_from_args();
+    match std::env::args().skip(1).next() {
+        Some(arg) => {
+            match arg.as_str() {
```

Comment:
Instead of having separate cases for separate sizes, just parse the input string as an integer. We don't know in advance which sizes we want. Also it's just bad practice to copy-paste code.